app/controllers/sessions_controller.rb

```diff
@@ -6,9 +6,10 @@ def create
 
     if user&.authenticate params[:session][:password]
       log_in user
-      redirect_to login_url
+      params[:session][:remember_me] == "1" ? remember(user) : forget(user)
+      redirect_to user
     else
-      flash.now[:danger] = t "invalid_login"
+      flash[:danger] = t "invalid_login"
```

Comment:
Sao lại xóa .now đi em
phân biệt giúp anh 2 cái flash này nhé

Reply:
rubocop em nó bị lỗi size for create is to high nên em xóa đi
flash: chuyển hướng trang, rồi hiển thị
flash.now: chỉ hiển thị mẫu

Reply:
em check lại xem tính năng hoạt động đúng ko nhé
Khi em nhập email or password lỗi thì có hiển thị dòng thông báo lỗi kia ko?

Reply:
![Screenshot from 2022-01-06 13-21-40](https://user-images.githubusercontent.com/89715376/148338117-a80de5d8-137f-41c7-a089-7594de75b282.png)
hoạt động bình thường ạ